src/compositor.rs

```diff
@@ -131,6 +133,11 @@ impl SurfaceData {
         self.scale_factor.load(Ordering::Relaxed)
     }
 
+    /// The scale factor of the output with the highest scale factor.
```

Comment:
I think this comment should say transform instead of scale factor?

Reply:
aye.